vwifi.c

```diff
@@ -24,6 +24,9 @@ MODULE_DESCRIPTION("virtual cfg80211 driver");
 #define NAME_PREFIX "vw"
 #define NDEV_NAME NAME_PREFIX "%d"
 
+#define VWIFI_WIPHY_NAME_LEN 12
+#define VWIFI_WIPHY_PREFIX "vwifi_phy"
```

Comment:
How about `vw_phy` for consistency?

Reply:
No problem, I'm preparing interview for summer internship recently.
I'll get it done as soon as possible.